internal/ingress/annotations/annotations_test.go

```diff
@@ -27,16 +27,20 @@ import (
 
 func TestIngressClassValidatorFunc(t *testing.T) {
 	tests := []struct {
-		ingress    string
-		controller string
-		isValid    bool
+		ingress      string
+		forceIngress bool
```

Comment:
Please rename `forceIngress` to something meaningful.
How about `SkipClasslessIngress`?